apex/transformer/pipeline_parallel/schedules/fwd_bwd_pipelining_without_interleaving.py

```diff
@@ -239,6 +240,7 @@ def forward_backward_pipelining_without_interleaving(
     deallocate_pipeline_outputs: bool = False,
     async_comm: bool = False,
     sequence_parallel_enabled: bool = False,
+    custom_sync_context_handler = None,
```

Comment:
Can you add some test case(s) with a certain context to https://github.com/NVIDIA/apex/blob/master/tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py#L283 once this gets "ready"?